homeassistant/components/open_meteo/weather.py

```diff
@@ -33,7 +37,9 @@ class OpenMeteoWeatherEntity(
 ):
     """Defines an Open-Meteo weather entity."""
 
-    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_native_precipitation_unit = LENGTH_MILLIMETERS
```

Comment:
I did not find documentation about what unit the open_meteo package uses for precipitation

Reply:
https://github.com/home-assistant/core/blob/03f0916e7cc9b1f0eded498c8d6f68f4cc26e5a8/homeassistant/components/open_meteo/__init__.py#L47

😉